specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -750,20 +806,414 @@
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/consumption/"
+        },
+        "x-ms-examples": {
+          "TransactionsListByCustomer": {
+            "$ref": "./examples/TransactionsListByCustomer.json"
+          },
+          "ReservationTransactionsListByCustomer": {
+            "$ref": "./examples/ReservationTransactionsListByCustomer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/billingAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/customerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "startDate",
+            "in": "query",
+            "description": "Start date",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "endDate",
+            "in": "query",
+            "description": "End date",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "$filter",
+            "description": "May be used to filter by transaction kind. The filter supports 'eq', 'lt', 'gt', 'le', 'ge', and 'and'. It does not currently support 'ne', 'or', or 'not'. Tag filter is a key value pair string where key and value is separated by a colon (:).",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/TransactionListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
     "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/departments": {
       "get": {
         "tags": [
           "Departments"
         ],
         "x-ms-examples": {
-          "DepartmentsListByBillingAccountName": {
-            "$ref": "./examples/DepartmentsListByBillingAccountName.json"
+          "DepartmentsListByBillingAccount": {
+            "$ref": "./examples/DepartmentsListByBillingAccount.json"
           },
-          "DepartmentsListByBillingAccountNameWithExpand": {
-            "$ref": "./examples/DepartmentsListByBillingAccountNameWithExpand.json"
+          "DepartmentsListByBillingAccountWithExpand": {
+            "$ref": "./examples/DepartmentsListByBillingAccountWithExpand.json"
           }
         },
-        "operationId": "Departments_ListByBillingAccountName",
+        "operationId": "Departments_ListByBillingAccount",
```

Comment:
this is SDK breaking change. Pls release  new service tag

Reply:
Undone the change for this specific API.